docs/validators/time.rst

```diff
@@ -0,0 +1,15 @@
+Time
+^^^^^^^^^^^^^^
+
+::
+
+    - do: time
+      age:  // validate based on the age of the PR
+        seconds: 86400
```

Comment:
As a user and someone configuring this, I would assume that one would be more likely to think in days than it is in seconds. Thoughts on using days instead of seconds?

Reply:
I also thought that but I wasn't sure if there is a use case for "hours" as well, if that is needed I thought seconds would give the most flexible option. If you feel that `days` is good enough as the lowest unit, I'll change to it.

Reply:
nvm  your comment below suggests that `days` is good enough, i'll change it